components/brave_wallet_ui/components/shared/add-custom-token-form/add-custom-token-form.tsx

```diff
@@ -107,12 +108,29 @@ export const AddCustomTokenForm = (props: Props) => {
   } = useTokenInfo(
     getBlockchainTokenInfo,
     combinedTokensList,
-    customAssetsNetwork || selectedNetwork
+    customAssetsNetwork
   )
   const {
     onAddCustomAsset
   } = useAssetManagement()
 
+  const { data: matchedCoingeckoId } = useGetCoingeckoIdQuery(
+    customAssetsNetwork
```

Comment:
we should still check if `tokenContractAddress` is truthy before initiating the query